src/KeyboardHandlerDropdown/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import LocalStorage from "../common/local-storage";
+
+import "./styles.scss"
+
+export default class KeyboardHandlerDropdown extends Component {
+  static propTypes = {
+    onKeyboardHandlerChange: PropTypes.func,
+  };
+
+  static defaultProps = {
+    onKeyboardHandlerChange: () => {},
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.dropdown = React.createRef();
+  }
+
+  componentDidMount = () => {
+    LocalStorage.get("keyboard_handler")
+      .then((keyboardHandler) => {
+        if (keyboardHandler) {
+          this.props.onKeyboardHandlerChange(keyboardHandler);
+          this.dropdown.current.value = keyboardHandler;
+        }
+      })
+      .catch(() => {});
+  };
+
+  handleChange = (e) => {
+    this.props.onKeyboardHandlerChange(e.target.value);
+    LocalStorage.set("keyboard_handler", e.target.value);
+  };
+
+  render() {
+    return (
+      <div className="keyboard-handler-dropdown">
+        <label htmlFor="keyboard-handler">Key bindings:</label>
```

Comment:
remove the label and just name the options "Default Keybindings", "Vim Keybindings" & "Emacs Keybindings". should make the UI feel less cluttered imo